test/config/PlayBasedServiceConfigurationSpec.scala

```diff
@@ -17,59 +17,66 @@
 package config
 
 import com.typesafe.config.ConfigFactory
-import model.{AllowedMimeTypes, ConsumingServicesConfiguration}
 import org.scalatest.{Assertions, GivenWhenThen, Matchers}
 import play.api.Configuration
 import uk.gov.hmrc.play.test.UnitSpec
 
 class PlayBasedServiceConfigurationSpec extends UnitSpec with Matchers with Assertions with GivenWhenThen {
-  "PlayBasedServiceConfiguration" should {
-    "parse consuming service configuration" in {
-      Given("a Play configuration file with configured services")
-      val configuration = Configuration(ConfigFactory.load("test-one.conf"))
 
-      When("the configuration is parsed")
-      val playBasedServiceConfiguration = new PlayBasedServiceConfiguration(configuration)
+  "PlayBasedServiceConfiguration" should {
 
```

Comment:
Do we need a case where a service specifies an empty mime-types? e.g.
`{ user-agent: "test-user-agent-one", mime-types: "" }`

We would seemingly apply this "as-is" in the current implementation, as this would take precedence over defaults.  Are we happy with this, or should we require at least one mime type to be specified, or should we treat this the same as if they had not provided an entry (and so fallback to the defaults)?

Reply:
I was anticipating that it would just block everything. This might not be good behavior though.... what do you reckon it should do?